src/applications/ivc-champva/10-7959f-2/tests/unit/config/submitTransformer.unit.spec.js

```diff
@@ -32,7 +32,6 @@ describe('submit transformer', () => {
     const newTransformData = JSON.parse(
       transformForSubmit(formConfig, formData),
     );
-    // eslint-disable-next-line no-console
-    expect(newTransformData.veteran.date_of_birth).to.equal('02/19/2004');
+    expect(newTransformData.veteran.date_of_birth).to.equal('2004-02-19');
```

Comment:
When I pulled stuff down to test for linting failures, this unit test also was now failing. I'm not sure if `transformForSubmit` was updated independently and this got through?

cc @reiting